### PR TITLE
Implement imu into localization basement testing patch 1

### PR DESCRIPTION
**Pull request by valentinschaeffer**
topic of odom data is missing, 
acceleration and deceleration limits are maybe not right,
defenitly, someone has to look over it

### CODE REVIEW

File: params/ekf.yaml [hunk 1]
```diff
@@ -164,15 +178,15 @@ ekf_filter_node:
...
-        deceleration_limits: [1.3, 0.0, 0.0, 0.0, 0.0, 4.5]
+        deceleration_limits: [3.9, 0.0, 0.0, 0.0, 0.0, 3.1]
 # If your robot cannot instantaneously reach its acceleration limit, the permitted change can be controlled with these
 # gains
-        acceleration_gains: [0.8, 0.0, 0.0, 0.0, 0.0, 0.9]
+        acceleration_gains: [0.5, 0.0, 0.0, 0.0, 0.0, 0.5]
 # If your robot cannot instantaneously reach its deceleration limit, the permitted change can be controlled with these
 # gains
-        deceleration_gains: [1.0, 0.0, 0.0, 0.0, 0.0, 1.0]
+        deceleration_gains: [0.5, 0.0, 0.0, 0.0, 0.0, 0.5]
```
Comment (liyuanlogi) [inline params/ekf.yaml:189]: Why did you change this values?

**Review by liyuanlogi — COMMENTED** [review]
Looks good and please see the comment.
And what do you mean topic of odom data is missing?

**Review by liyuanlogi — APPROVED** [review]

File: params/ekf.yaml (hunk 1 above)
Comment (valentinschaeffer) [reply params/ekf.yaml:189]: I calculated the values ​​roughly for the AGV there. Im not sure if its necessary

Comment (liyuanlogi) [reply params/ekf.yaml:189]: If you want to compare the accuracy of odometry with and without imu data, then maybe we should keep all other parameters the same? So other parameters won't affect the result?
